### PR TITLE
refactor(p-dict-list): delete PDictList and replace it with PTag

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console

### Description
스크린샷 2022-10-26 오후 3 21 30

- The utilization of `PDictList` was low, so delete it and replaced with `PTag`. (`PDictList`의 활용도가 낮아, 이를 삭제하고 `PTag`로 대체함)
- Tested on Console and Storybook.

### CODE REVIEW

File: src/data-display/dynamic/dynamic-field/templates/dict/index.vue [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
+        <template v-else>
+            <p-tag v-for="([objKey, objValue], idx) in Object.entries(dictData)"
+                   :key="`tag-${idx}-${objKey}`"
+                   :key-item="{ name: objKey, label: objKey }"
+                   :value-item="{ name: objValue, label: objValue }"
+                   :deletable="false"
+            />
+        </template>
+    </span>
+</template>
```
Comment (yuda110) [inline src/data-display/dynamic/dynamic-field/templates/dict/index.vue:20]: Change from functional to single file component.

File: src/data-display/dynamic/dynamic-field/templates/dict/index.vue [hunk 2]
```diff
@@ -1,13 +1,35 @@
...
 <script lang="ts">
-import type { DictDynamicFieldProps } from '@/data-display/dynamic/dynamic-field/templates/dict/type';
-import PDictList from '@/data-display/dynamic/dynamic-field/templates/list/dict-list/PDictList.vue';
-import type { TextOptions } from '@/data-display/dynamic/dynamic-field/type/field-schema';
+import { computed, reactive, toRefs } from 'vue';
+
+import PTag from '@/data-display/tags/PTag.vue';
 import PAnchor from '@/inputs/anchors/PAnchor.vue';
 
 export default {
```
Comment (sulmoJ) [inline src/data-display/dynamic/dynamic-field/templates/dict/index.vue]: It would be nice to add defineComponent to this code!

**Review by sulmoJ — APPROVED** [review]
LGTM!!
I leave a small comment!

**Review by song2kim — APPROVED** [review]
LGTM

**Pull request merged**